Code/Tools/SceneAPI/SceneCore/Tests/Utilities/CoordinateSystemConverterTests.cpp

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzTest/AzTest.h>
+#include <AzCore/Math/Matrix3x4.h>
+#include <SceneAPI/SceneCore/Utilities/CoordinateSystemConverter.h>
+
+namespace EMotionFX
+{
+    const AZ::Vector3 sourceBasisVectors[3] = {
+        AZ::Vector3(1.0f, 0.0f, 0.0f),
+        AZ::Vector3(0.0f, 1.0f, 0.0f),
+        AZ::Vector3(0.0f, 0.0f, 1.0f),
+    };
+    const AZ::Vector3 targetBasisVectors[3] = {
+        AZ::Vector3(-1.0f, 0.0f, 0.0f),
+        AZ::Vector3(0.0f, 1.0f, 0.0f),
+        AZ::Vector3(0.0f, 0.0f, -1.0f),
+    };
+    const AZ::u32 targetBasisIndices[3] = { 0, 1, 2 };
+
+    TEST(CoordinateSystemConverterTests, TransformsCorrectlyCreatedFromBasisVectors)
+    {
+        AZ::SceneAPI::CoordinateSystemConverter coordinateSystemConverter =
+            AZ::SceneAPI::CoordinateSystemConverter::CreateFromBasisVectors(sourceBasisVectors, targetBasisVectors, targetBasisIndices);
+
+        EXPECT_TRUE(coordinateSystemConverter.GetSourceTransform().GetBasisX().IsClose(sourceBasisVectors[0]));
```

Comment:
nitpick - if you use EXPECT_THAT with the IsClose matcher I think you will get more useful output if the test fails